src/main/java/io/confluent/connect/hdfs/HdfsSinkTask.java

```diff
@@ -93,6 +94,9 @@ public void start(Map<String, String> props) {
         hdfsWriter.stop();
       }
     }
+
+    log.info("Consumer offsets will be obtained from filenames on HDFS. This connector will not "
+        + "commit or use any offsets from the __consumer_offsets topic.");
```

Comment:
nit: the ``or use`` part is not accurate. The connector will use consumer offsets from that topic iff files are not present in hdfs and such offset exists for this consumer group. 

``This connector will not commit any offsets to the '__consumer_offsets' topic.`` might be what we want.

Reply:
Yeah, I like @kkonstantine's suggestion. @wicknicks, it's a minor change but would be nice to have.

Reply:
So we deliberately put the "use any offsets" part to simplify the understanding on where the offsets come from. Apparently, offsets management in HDFS has been confusing in the past. @rhauch?